doc/api/http.md

```diff
@@ -2119,7 +2119,7 @@ http.get('http://nodejs.org/dist/index.json', (res) => {
   const contentType = res.headers['content-type'];
 
   let error;
-  if (statusCode !== 200) {
+  if (!(statusCode >= 200 && statusCode <= 206)) {
```

Comment:
I think the original is better. If that endpoint is expected to return 200 it is better to be strict. Also 201 is not usually used for a GET request, 204 has no content, etc.